include/ppx/xr_component.h

```diff
@@ -118,11 +129,13 @@ class XrComponent
 public:
     virtual ~XrComponent() = default;
 
-    void InitializeBeforeGrfxDeviceInit(const XrComponentCreateInfo& createInfo);
-    void InitializeAfterGrfxDeviceInit(const grfx::InstancePtr pGrfxInstance);
-    void Destroy();
+    void     InitializeBeforeGrfxDeviceInit(const XrComponentCreateInfo& createInfo);
+    void     InitializeAfterGrfxDeviceInit(const grfx::InstancePtr pGrfxInstance);
+    XrResult InitializeInteractionProfile();
```

Comment:
Could you add some comments for the new functions/variables? 
For example, here mention what you have in the function body: "// Controller support is not essential, so if anything fail, we just return."

Reply:
The function does have a status return. So the comment is at call site.